src/bin/vip-app.js

```diff
@@ -19,7 +19,7 @@ command( { requiredArgs: 1, format: true } )
 		let res;
 
 		try {
-			res = await app( arg[ 0 ], 'id,name,environments{id,name,defaultDomain}' );
+			res = await app( arg[ 0 ], 'id,environments{name,datacenter,branch,currentCommit,defaultDomain}' );
```

Comment:
- `datacenter` isn't very useful.
- `branch` is a good idea but not as useful without the repo.
- `defaultDomain` should probably be renamed on display to primary domain?

Reply:
`currentCommit` is displaying as `[object Object]` for me.

Reply:
There’s a PR to fix the currentCommit https://github.com/Automattic/vip-go-api-public/pull/86

Good point about branch. I was also thinking about this in the context of app names — we don’t show the app name anywhere in vip app <app>. Maybe not quite as bad though; presumably you know what app you’ve specified?